scripts/bfasst/compare/conformal.py

```diff
@@ -74,6 +74,19 @@ def compare_netlists(self, design, print_to_stdout=True):
         # Create do file
         do_file_path = self.create_do_file(design)
 
+        # Create remote machine folders
+        cmd = (
+            "mkdir -p bfasst_libs;"
+            + "mkdir -p bfasst_libs/xilinx;"
+            + "touch bfasst_libs/xilinx/cells_sim.v;"
```

Comment:
I don't think you need this touch command.  Can you verify it works without it?

Reply:
Both work without the touch command.